### PR TITLE
Release  v9.1.3

### PR DESCRIPTION
**Pull request by RostislavKreisinger**
| Q             | A
| ------------- | ---
|Description, reason for the PR| Release framework v9.1.3
|New feature| Yes 
|[BC breaks](https://docs.shopsys.com/en/latest/contributing/backward-compatibility-promise/)| No 
|Fixes issues| ... 
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

### CODE REVIEW

File: utils/releaser/src/GithubActions/GithubActionsStatusReporter.php [hunk 1]
```diff
@@ -33,6 +34,9 @@ final class TravisStatusReporter
         'shopsys/sysconfig',
         'shopsys/sysreports',
         'shopsys/sysstdlib',
+        'shopsys/backend-api',
+        'shopsys/phpstorm-inspect',
+        'shopsys/product-feed-interface',
```
Comment (TomasLudvik) [inline utils/releaser/src/GithubActions/GithubActionsStatusReporter.php:39]: I think this package should be part of the check, right?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,126 @@
...
+## [v10.0.2](https://github.com/shopsys/shopsys/tree/v10.0.2) (2022-07-20)
+
+[Full Changelog](https://github.com/shopsys/shopsys/compare/v10.0.1...v10.0.2)
+
+**Merged pull requests:**
+
+- \[shopsys\] release of v10.0.2 [\#2489](https://github.com/shopsys/shopsys/pull/2489) ([grossmannmartin](https://github.com/grossmannmartin))
+
+## [v10.0.1](https://github.com/shopsys/shopsys/tree/v10.0.1) (2022-06-16)
+
```
Comment (TomasLudvik) [inline CHANGELOG.md:42]: I do not think that versions higher than 9.1.3 should be present here

File: upgrade/UPGRADE-v9.1.3.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
-# [Upgrade from v9.1.2 to v9.1.3-dev](https://github.com/shopsys/shopsys/compare/v9.1.2...9.1)
+# [Upgrade from v9.1.2 to v9.1.3](https://github.com/shopsys/shopsys/compare/v9.1.2...9.1)
 
-This guide contains instructions to upgrade from version v9.1.2 to v9.1.3-dev.
+This guide contains instructions to upgrade from version v9.1.2 to v9.1.3.
```
Comment (TomasLudvik) [inline upgrade/UPGRADE-v9.1.3.md:3]: In this file there is several upgrade notes for changes not mentioned in changelog

**Review by TomasLudvik — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,120 +4,18 @@
 
 [Full Changelog](https://github.com/shopsys/shopsys/compare/v10.0.4...v9.1.3)
 
-**Merged pull requests:**
-
-- \[framework\] update heureka/overeno-zakazniky to latest version 4.0.1 [\#2534](https://github.com/shopsys/shopsys/pull/2534) ([RostislavKreisinger](https://github.com/RostislavKreisinger))
```
Comment (TomasLudvik) [inline CHANGELOG.md]: I asked you to fill missing changes in 9.1.3 and you even removed the only one mentioned :D

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,24 @@
...
+
+- composer updates [\#2397](https://github.com/shopsys/shopsys/pull/2397) ([vitek-rostislav](https://github.com/vitek-rostislav))
+
+**Merged pull requests:**
+
+- \[framework\] update heureka/overeno-zakazniky to latest version 4.0.1 [\#2534](https://github.com/shopsys/shopsys/pull/2534) ([RostislavKreisinger](https://github.com/RostislavKreisinger))
+
+
+# Changelog
+
```
Comment (TomasLudvik) [inline CHANGELOG.md:21]:
```suggestion
```

Comment (RostislavKreisinger) [reply CHANGELOG.md]: ok

**Review by TomasLudvik — APPROVED** [review]

**Pull request merged**